kubernetes/hypertrace.sh

```diff
@@ -116,11 +116,18 @@ case "$subcommand" in
 
     echo "[INFO] installing hypertrace data services. namespace: ${HT_KUBE_NAMESPACE}, context: ${HT_KUBE_CONTEXT}"
     helm dependency update ${HYPERTRACE_HOME}/data-services ${HELM_FLAGS}
-    helm upgrade hypertrace-data-services ${HYPERTRACE_HOME}/data-services -f ${HYPERTRACE_HOME}/data-services/values.yaml -f ${HYPERTRACE_HOME}/clusters/$HT_PROFILE/values.yaml --install --wait ${HELM_FLAGS} --timeout ${HT_INSTALL_TIMEOUT}m --set htEnv=${HT_ENV}
-
+    if [ $HT_DATA_STORE == "postgres" ]; then
```

Comment:
Can we do something like this for ease of reading?

```
values_list = "${HYPERTRACE_HOME}/data-services -f ${HYPERTRACE_HOME}/data-services/values.yaml"
if [ $HT_DATA_STORE == "postgres" ]; then
values_list += "-f ${HYPERTRACE_HOME}/data-services/postgres/values.yaml"
fi
values_list += "${HYPERTRACE_HOME}/clusters/$HT_PROFILE/values.yaml"

helm upgrade hypertrace-data-services ${values_list} --install --wait ${HELM_FLAGS} --timeout ${HT_INSTALL_TIMEOUT}m --set htEnv=${HT_ENV}
```

Reply:
This will make script smart. The purpose of having command this way since beginning was people can copy paste these commands if they don't want to install using script for some reason. For eg, in case of windows deployment you have to use commands instead of script as windows can't run bash scripts.